api.planx.uk/inviteToPay/paymentRequest.ts

```diff
@@ -128,33 +128,6 @@ export const fetchPaymentRequestViaProxy = fetchPaymentViaProxyWithCallback(
   },
 );
 
-export const addGovPayPaymentIdToPaymentRequest = async (
```

Comment:
I needed to move this into a shared `utils.ts` as I was hitting an issue with circular dependencies on the build application.